deploy/charts/csi-driver-spiffe/Chart.yaml

```diff
@@ -9,5 +9,5 @@ maintainers:
     url: https://cert-manager.io
 sources:
 - https://github.com/cert-manager/csi-driver-spiffe
-appVersion: v0.3.0
-version: v0.3.0
+appVersion: v0.3.1
+version: v0.3.1
```

Comment:
suggestion: Would it be possible for you to leave this as-is? I'll bump it later when I do a release, and I prefer to do the bump just before the release.

(I'm also not sure if I'll do 0.3.1 or 0.4.0 yet!)

Reply:
yeah, sorry: I thought it might be the reason why the `README.md` check was failing. Will revert. Realized, that you use `helm-docs` for README.md update. Will fix as soon as I have downloaded the tool (I am on a spotty train wifi ;) )